sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/SequencePoint.java

```diff
@@ -51,4 +55,8 @@ public int getLineEnd() {
   public int getHits() {
     return hits;
   }
+
+  public String getCoverageKey() {
```

Comment:
I would name it `getUniqueIdentifier` to make it obvious that it identifies the SequencePoint